.gitignore

```diff
@@ -41,7 +41,6 @@ DerivedData
 # CocoaPods
 #
 Pods/
-*Podfile.lock
```

Comment:
https://guides.cocoapods.org/using/using-cocoapods#benefits-of-ignoring-the-pods-directory

> Whether or not you check in the Pods directory, the Podfile and **Podfile.lock** should always be kept under version control.

Reply:
Agreed :)